avocado/plugins/replay.py

```diff
@@ -206,18 +212,25 @@ def run(self, args):
                      "--replay-ignore.")
         else:
             if getattr(args, 'multiplex', None) is not None:
+                # Disable the mux from the original job
                 log.warn('Overriding the replay multiplex with '
-                         '--multiplex-file.')
-                # Use absolute paths to avoid problems with os.chdir
-                args.multiplex = [os.path.abspath(_)
-                                  for _ in args.multiplex]
+                         '--multiplex-file. Note that currently '
+                         'this ignores the system-wide mux options '
+                         'so it really uses only the values you provide '
+                         'on the cmdline.')
             else:
                 mux = jobdata.retrieve_mux(resultsdir)
                 if mux is None:
                     log.error('Source job multiplex data not found. Aborting.')
                     sys.exit(exit_codes.AVOCADO_JOB_FAIL)
                 else:
+                    # Ignore data manipulation. This is necessary, because
```

Comment:
Not really, as there is no plugins order there is no way to avoid following plugins trying to alter Mux. This is one way to make sure it won't happen.